Content for snooze page (#914)
novuhq/docs: content/docs/platform/inbox/configuration/snooze.mdx

---
pageTitle: 'Snooze notifications'
title: 'Snooze'
description: 'Allow users to temporarily hide notifications and resurface them later using built-in snooze functionality in the Inbox component.'
icon: 'Clock'
---

> The `<Inbox />` component includes built-in support for snoozing notifications, which can be used to temporarily dismiss notifications and have them reappear at a more convenient time.

Reviewer: ```suggestion
The `<Inbox />` component includes built-in support for snoozing the notifications, allowing you to temporarily dismiss them and have them reappear at a more convenient time.
```

Author: @jainpawan21 This isn't correct. We don't use "Allow". Kindly leave it as it is.

## How to snooze a notification

All dates and times are interpreted in your subscriber's local timezone. They can snooze a notification directly from the Inbox using one of the following options:

- Preset options:
  - An hour from now
  - Tomorrow
- Custom date and time:
  - A built-in date and time picker allows users to select a specific future time.

![Snooze a notification](/images/inbox/snooze-notification.png)

The minimium snooze time must be at least 3 minutes in the future. While the maximum duration a notification can be snoozed depends on your cloud workspace plan:
- **Free**: Up to 24 hours per notification
- **Trial**: Up to 90 days during the 14-day trial, same as Pro
- **Pro / Team**: Up to 90 days
- **Enterprise**: Defaults to 90 days, extendable on request

While the Inbox component handles snoozing automatically, the [JavaScript SDK](/platform/sdks/javascript#methods) also exposes `.snooze()` and `.unsnooze()` methods on the notification object to build custom snooze functionality.

This let's you create custom UI elements or workflows that trigger snooze actions outside of the default dropdown menu.

## Managing snoozed notifications

> The Inbox component includes a dedicated Snoozed tab that automatically displays notifications scheduled to reappear later. From this tab, users can review all snoozed notifications and take further actions as needed.

Reviewer: ```suggestion
The Inbox component includes a dedicated **Snoozed** tab that automatically displays notifications scheduled to reappear later. From this tab, users can review all snoozed notifications and take further actions as needed.
```
